README.md

```diff
@@ -1,5 +1,6 @@
 # ckanext-datajson
 
+![Github Actions](https://github.com/GSA/ckanext-datajson/actions/workflows/test.yml/badge.svg)
```

Comment:
Might want to wrap it in a link to go look at the appropriate workflow, like the CircleCI badge does...